courses/views.py

```diff
@@ -38,6 +41,44 @@ class ProgramViewSet(viewsets.ReadOnlyModelViewSet):
     serializer_class = ProgramSerializer
 
 
+class ProgramLearnersView(APIView):
```

Comment:
oh, I don't really know, it just helped me out in the past so I thought I would mention it!